transport_versioned_docs/version-1.0.0/workflow-client-server-secure.md

```diff
@@ -0,0 +1,147 @@
+
+Finally, using the different files generated, we can create the certificate file the server will use to authenticate itself : 
+```shell
+openssl.exe x509 -req -in myGameServerCertificateSigningRequest.csr -CA myGameClientCA.pem -CAkey clientPrivateKeyForRootCA.key -CAcreateserial -out myGameServerCertificate.crt -days 365 -sha256
+```
+You should have now generated a total of five files. Out of these, only will be used later on : 
+* The content of the `myGameClientCA.pem` file, will be used client side as the `pem` parameter.
+* On the server end, `myGameServerPrivate.key` file content will be used for the `rsaKey` parameter.
+* `myGameServerCertificate.crt` will be used by the `rsa` parameter on the server end. 
+
+### Boiler Plate file holding the secure parameters
+Create a `SecureParameters.cs` script file to hold your the certificates and the private key. Place it in the same folder as the minimal server and minimal client scripts.
+Start by enabling Unity TLS with : 
+```cs 
+#define ENABLE_MANAGED_UNITYTLS
+```
+and add the following dependencies: 
+```cs
+using Unity.Collections;
+using Unity.Networking.Transport.TLS;
+```
+Then declare the secureParameters class and the boilerplate code that will hold your secure information:
+```cs
+public static class SecureParameters
+{
+        public static FixedString32Bytes commonName = new FixedString32Bytes("common-name");  // Use the common name you used to define the server certificate. 
+        public static FixedString4096Bytes myGameClientCA = new FixedString4096Bytes(
+        @"-----BEGIN CERTIFICATE-----
+                     ---                  
+          -----END CERTIFICATE-----"); // This should contain the content of myGameClientCA.pem 
+        public static FixedString4096Bytes myGameServerCertificate = new FixedString4096Bytes(
+        @"-----BEGIN CERTIFICATE-----
+                    ---
+          -----END CERTIFICATE-----"); // This should contain the content of myGameServerCertificate.pem  
+        public static FixedString4096Bytes myGameServerPrivate = new FixedString4096Bytes(
+        @"-----BEGIN RSA PRIVATE KEY-----
+                       ---
+          -----END RSA PRIVATE KEY-----"); // This should contain the content of myGameServerPrivate.key  
+}
+```  
+
+### Creating the Secure Server
+
+
+Starting from the minimal server sample code, create a `NetworkSettings` object.
+
+```cs
+private NetworkSettings settings = new NetworkSettings();
+```
+
+Within the `start()` method, configure this `NetworkSettings` as following: 
+```cs
+void Start ()
+    {
+        settings.WithSecureParameters(
+            hostname: ref SecureParameters.commonName,                    // Use the common name you used to generate `myGameServerCertificate.crt`
+            sslHandshakeTimeoutMin: 500,    // Duration in ms
+            sslHandshakeTimeoutMax: 2000,   // Duration in ms
```

Comment:
Can leave those out, the default values should be fine in most cases.

Reply:
removed